src/index.html

```diff
@@ -104,19 +104,24 @@ <h4 class="opacity-60">GitHub Discussions</h4>
         <div class="max-width-container">
           <div class="sm:flex sm:flex-row gap-10">
             <div class="basis-1/2 my-auto">
-              <h1 class="text-white mb-6">
+              <h2 class="text-white mb-6">
```

Comment:
This should only shrink on mobile and small screens. I initially only said mobile in my first review but after implementing it, it looks better as a smaller font size on small screens, too.

```suggestion
              <h1 class="text-2xl text-white mb-6 md:text-3xl">
```